src/org/omegat/util/TMXReader2.java

```diff
@@ -160,11 +160,12 @@ public void readTMX(File file, final Language sourceLanguage, final Language tar
                     if ("tu".equals(eStart.getName().getLocalPart())) {
                         parseTu(eStart);
                         ParsedTuv origTuv = getTuvByLang(sourceLanguage);
-                        // This ParsedTuv for the target translation is not used, but is there to avoid
-                        // any NPE during the callback.onEntry below.
+
+                        // Find the best matching target Tuv.
+                        // This targetTuv is used to keep the LoadCallback interface compatibility with
+                        // current tests and other implementations.
```

Comment:
I think we don't need this comment either. `ExternalTMFactory` is just one client, and it's not even the main one (I would consider that to be the one in `ProjectTMX`), so the fact that it happens to ignore `targetTuv` is not something to point out here.

Reply:
Comment removed